tests/baselines/reference/asyncMethodWithSuperConflict_es6.js

```diff
@@ -87,23 +88,24 @@ class B extends A {
             const cache = Object.create(null);
             return name => cache[name] || (cache[name] = { get value() { return geti(name); }, set value(v) { seti(name, v); } });
         })(name => super[name], (name, value) => super[name] = value);
+        const _super_x = { get value() { return super.x; }, set value(v) { super.x = v; } };
         return __awaiter(this, void 0, void 0, function* () {
             const _super = null;
             const f = () => { };
             // call with property access
-            _super_1("x").value.call(this);
+            _super_x().value.call(this);

```

Comment:
Maybe I'm missing something, but the way `_super_x` is defined in line 91 (an object), shouldn't this be 
```ts
_super_x.value.call(this);
//      ^ no parentheses
```
instead?

Reply:
Good catch, thanks!

I've fixed this by moving to the implementation that @rbuckton suggested in #21088, i.e. creating a single object `_superProps` that contains configured properties for each named property.